docs/en/userguide/install.md

```diff
@@ -143,8 +189,8 @@ $ kubectl delete ns fluid-system
 
 ### Advanced Configuration
 
-In some cloud vendors, the default mount root directory `/alluxio-mnt` is not writable, so you have to modify the directory location
+In some cloud vendors, the default mount root directory `/runtime-mnt` is not writable, so you have to modify the directory location
```

Comment:
how about add this to faq, just a suggestion.

Reply:
> how about add this to faq, just a suggestion.

I think it's an important matter needing attention because lots of users will meet this case, it's better to put it in installing doc.